psqlgraph/mocks.py

```diff
@@ -205,8 +222,9 @@ def create(self, label, override=None, all_props=False):
                 continue
 
             try:
-                override_val = self.get_global_value(prop) or \
-                    override.get(prop)
+                override_val = self.validate_override_value(
```

Comment:
wouldn't `override.get(prop) or self.get_global_value(prop)` work? don't quite see the reason in adding an extra layer of validation, where the validation is already happening on the property factory level?

Reply:
synched up offline. looks reasonable